src/test/update-items.test.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable linebreak-style */
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable linebreak-style */
+/* eslint quotes: ["error", "double", { "avoidEscape": true }] */
```

Comment:
[OPTIONAL] If I were you, I wouldn't disable linters for they are there to help us align to best practice which results in better code quality. They help ensure standard way of code writing amongst team members. Moreover, you might not have the luxury of disabling them in the real world. Kindly try to abide by the rules set in the linter configurations.

Reply:
Thank you, Tekle, I will fix this soon,